src/main/java/lotto/domain/model/lotto/LottoTicket.java

```diff
@@ -14,20 +14,36 @@ public LottoTicket(final Set<LottoNumber> numbers) {
 
     private void validate(final Set<LottoNumber> numbers) {
         if (numbers == null || numbers.size() != TICKET_NUMBER_COUNT) {
-            final String EXCEPTION_MESSAGE_FORMAT = "로또 번호는 반드시 %d개의 숫자로 구성되어야 합니다.";
-
-            throw new IllegalArgumentException(String.format(EXCEPTION_MESSAGE_FORMAT, TICKET_NUMBER_COUNT));
+            throw new IllegalArgumentException(
+                    String.format("로또 번호는 반드시 %d개의 숫자로 구성되어야 합니다.", TICKET_NUMBER_COUNT));
         }
     }
 
     public boolean contains(final LottoNumber number) {
         return numbers.contains(number);
     }
 
+    public int countMatchingNumbers(final LottoTicket other) {
+        return (int) numbers.stream().filter(other::contains).count();
```

Comment:
```suggestion
        return (int) numbers.stream().
        filter(other::contains)
        .count();
```
이런식으로 작성한다면 훨씬 위에서 아래로 읽히니 좋을수 있습니다!